llvm/lib/Transforms/Yk/StackMaps.cpp

```diff
@@ -83,6 +83,7 @@ class YkStackmaps : public ModulePass {
                       ArrayRef<Value *>(Args));
       Count++;
     }
+    M.dump();
```

Comment:
Left-over debugging :)

Reply:
Ok to rebase for this, as this is only present in the second commit.

Reply:
Yep, kill the memset at the same time.

Reply:
No rebasing please: fresh commit always then squash at the end.

Reply:
Oh, I see now: it's removed in the third commit.

Reply:
Fixed in [71545c7](https://github.com/ykjit/ykllvm/pull/52/commits/71545c7f1ae1d4a54a6d937471eed5467afd9f89) (required rebase)